src/scales/scale.linear.js

```diff
@@ -121,6 +146,12 @@ module.exports = function(Chart) {
 				});
 			}
 
+			// Take a snapshot of scale limit to be used if invalid values for { max, min }
+			me.takeScaleLimitSnapShot(me.min, me.max);
```

Comment:
Will this work if the chart is created with all datasets hidden?

ie, adding `hidden: true` to each dataset on chart creation